packages/flutter/lib/src/widgets/scroll_absolute.dart

```diff
@@ -0,0 +1,678 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math' as math;
+import 'dart:ui' as ui show window;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/physics.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/scheduler.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+import 'gesture_detector.dart';
+import 'overscroll_indicator.dart';
+import 'scroll_simulation.dart';
+import 'notification_listener.dart';
+import 'scroll_notification.dart';
+import 'scrollable.dart';
+
+/// Scrolling logic delegate for lists and other unremarkable scrollable
+/// viewports.
+///
+/// See also:
+///
+/// * [BouncingAbsoluteScrollPositionMixIn], which is used by this class to
+///   implement the scroll behavior for iOS.
+/// * [ClampingAbsoluteScrollPositionMixIn] and [GlowingOverscrollIndicator],
+///   which are used by this class to implement the scroll behavior for Android.
+class ViewportScrollBehavior extends ScrollBehavior2 {
+  ViewportScrollBehavior({
+    Tolerance scrollTolerances,
+  }) : scrollTolerances = scrollTolerances ?? defaultScrollTolerances;
+
+  // TODO(ianh): in the material layer, we should introduce a
+  // ScrollConfiguration that introduces a version of this that overrides
+  // platform and color and gets the values from the Theme.
+
+  /// The accuracy to which scrolling is computed.
+  ///
+  /// Defaults to [defaultScrollTolerances].
+  final Tolerance scrollTolerances;
+
+  /// The accuracy to which scrolling is computed by default.
+  ///
+  /// This is the default value for [scrollTolerances].
+  static final Tolerance defaultScrollTolerances = new Tolerance(
+    // TODO(ianh): Handle the case of the device pixel ratio changing.
+    // TODO(ianh): Get this from the local MediaQuery not dart:ui's window object.
+    velocity: 1.0 / (0.050 * ui.window.devicePixelRatio), // logical pixels per second
+    distance: 1.0 / ui.window.devicePixelRatio // logical pixels
+  );
+
+  /// The platform whose scroll physics should be implemented.
+  ///
+  /// Defaults to the current platform.
+  TargetPlatform getPlatform(BuildContext context) => defaultTargetPlatform;
+
+  /// The color to use for the glow effect whet [platform] indicates a platform
+  /// that uses a [GlowingOverscrollIndicator].
+  ///
+  /// Defaults to white.
+  Color getAccentColor(BuildContext context) => const Color(0xFFFFFFFF);
```

Comment:
AccentColor is a material concept...  Maybe a more specific name?  This is used for the glow, right?  getThemeColor ?